CareKitStore/CareKitStoreTests/OCKHealthKitStore/TestHealthKitPassthroughStore+Events.swift

```diff
@@ -129,6 +130,8 @@ class TestHealthKitPassthroughStoreEvents: XCTestCase {
             case heartRateTask.id:
                 XCTAssertEqual(outcomeValues.count, 2)
                 XCTAssertEqual(outcomeValues.first?.doubleValue, 70)
+                XCTAssertEqual(outcomeValues.first?.startDate, heartRateStart)
+                XCTAssertEqual(outcomeValues.first?.endDate, heartRateEnd)
```

Comment:
Test to check if the sample's startDate and endDate were successfully saved in their OCKOutcomeValue